packages/editor/src/components/provider/index.js

```diff
@@ -273,6 +274,7 @@ export const ExperimentalEditorProvider = withRegistryProvider(
 									{ type === 'wp_navigation' && (
 										<NavigationBlockEditingMode />
 									) }
+									<EditorKeyboardShortcuts />
```

Comment:
No, because we want the shortcuts to be registered regardless of whether they are actually applied. Registering the shortcuts allow them to be visible in the keyboard shortcuts modal.

Reply:
I still don't get how it would affect the modal if we moved them in Editor provider. Will do some testing to verify 😄

Reply:
If we ever reconsider, I think we should just unify the components/registration...